### ISSUE TITLE
Pcie Nvme storages not reported

### ISSUE THREAD

**Issue by armageddon51** [lpereira/hardinfo#529]
It doesn't seem that the program can pick up pcie nvme storage devices. I have two of those and none is appearing under peripherals/storage. Can it be implemented ?
Tx

>inxi -d
Drives:    Local Storage: total: 1.84 TiB used: 1.05 TiB (57.1%) 
ID-1: /dev/nvme0n1 vendor: HP model: SSD EX920 256GB size: 238.47 GiB 
ID-2: /dev/nvme1n1 vendor: A-Data model: SX8200PNP size: 238.47 GiB 
.....

One of those is appearing under pci peripherals ;

0000:25:00.0 : ADATA Technology Co., Ltd. XPG SX8200 Pro PCIe Gen3x4 M.2 2280 Solid State Drive

**Comment by bp0** [comment]
It looks like you're using an older hardinfo. See #163. You'd need to use the version from git to see it.
Will look something like this:
![Screenshot_2020-01-02_12-09-44](https://user-images.githubusercontent.com/1758090/71683923-7a92c600-2d59-11ea-8fcf-17f33acf976f.png)

**Comment by armageddon51** [comment]
Hello the version I got is from Manjaro AUR and is version 0.6-alpha (compile on Oct. 2018). I don't see a newer one on github which has 0.5.1 Alpha ?? What version is the latest please ?
Thanks

**Comment by bp0** [comment]
Yeah, you'd need one from after Jan 2019. You must build from git master. I know it is confusing right now, but the version number doesn't really mean anything. Last tagged in 2009. @lpereira wants gtk3 to be solid before tagging a new one, of course he's the one that broke the GTK3 build (#410) :slightly_smiling_face:

**Comment by armageddon51** [comment]
Hi again, after a couple of iterations, I found that there is also in the Manjaro AUR, a "hardinfo-git" version to be build. I use this version and yes, it does recognize the nvme drives as you said. It is still version 0.6alpha though but I ask the maintainer to recompile it. 

one note : it always ask the permission to load the "computer.so" module which obviously is Yes. Do we really need that question ?
Tx
